packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -0,0 +1,126 @@
+import { randomBytes } from "crypto";
+
+import { sendTeamInviteEmail } from "@calcom/emails";
+import { updateQuantitySubscriptionFromStripe } from "@calcom/features/ee/teams/lib/payments";
+import { IS_TEAM_BILLING_ENABLED, WEBAPP_URL } from "@calcom/lib/constants";
+import { getTranslation } from "@calcom/lib/server/i18n";
+import { prisma } from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import { isEmail } from "../util";
+import type { TInviteMemberInputSchema } from "./inviteMember.schema";
+import {
+  checkPermissions,
+  getTeamOrThrow,
+  getEmailsToInvite,
+  getUserToInviteOrThrowIfExists,
+  checkInputEmailIsValid,
+  getOrgConnectionInfo,
+  createNewUserConnectToOrgIfExists,
+  throwIfInviteIsToOrgAndUserExists,
+  createProvisionalMembership,
+  getIsOrgVerified,
+  sendVerificationEmail,
+} from "./utils";
+
+type InviteMemberOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TInviteMemberInputSchema;
+};
+
+export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) => {
+  const team = await getTeamOrThrow(input.teamId, input.isOrg);
```

Comment:
We shouldn't accept isOrg as a request input, it can be intercepted and changed.

Reply:
We can fix it as a followup. This might or might not be a security issue at the moment.